aux/javapackages-tools.spec

```diff
@@ -19,133 +34,141 @@ Source0:        javapackages-%{version}.tar.gz
+Requires:       %{?scl_prefix}xmvn-minimal >= 2
+Requires:       %{?scl_prefix}xmvn-mojo >= 2
+Requires:       %{?scl_prefix}xmvn-connector-aether >= 2
 # Common Maven plugins required by almost every build. It wouldn't make
 # sense to explicitly require them in every package built with Maven.
-Requires:       maven-compiler-plugin
-Requires:       maven-jar-plugin
-Requires:       maven-resources-plugin
-Requires:       maven-surefire-plugin
+Requires:       %{?scl_prefix}maven-compiler-plugin
+Requires:       %{?scl_prefix}maven-jar-plugin
+Requires:       %{?scl_prefix}maven-resources-plugin
+Requires:       %{?scl_prefix}maven-surefire-plugin
 # Tests based on JUnit are very common and JUnit itself is small.
 # Include JUnit provider for Surefire just for convenience.
-Requires:       maven-surefire-provider-junit
+Requires:       %{?scl_prefix}maven-surefire-provider-junit
 # testng is quite common as well
-Requires:       maven-surefire-provider-testng
+Requires:       %{?scl_prefix}maven-surefire-provider-testng
 
-%description -n maven-local
+%description -n %{?scl_prefix}maven-local
 This package provides macros and scripts to support packaging Maven artifacts.
 
-%package -n gradle-local
+%package -n %{?scl_prefix}gradle-local
 Summary:        Local mode for Gradle
 Requires:       %{name} = %{version}-%{release}
-Requires:       javapackages-local = %{version}-%{release}
-Requires:       gradle >= 2.2.1-2
-Requires:       xmvn-connector-gradle >= 2
+Requires:       %{?scl_prefix}javapackages-local = %{version}-%{release}
+Requires:       %{?scl_prefix}gradle >= 2.2.1-2
+Requires:       %{?scl_prefix}xmvn-connector-gradle >= 2
 
-%description -n gradle-local
+%description -n %{?scl_prefix}gradle-local
 This package implements local mode for Gradle, which allows artifact
 resolution using XMvn resolver.
 
-%package -n ivy-local
+%package -n %{?scl_prefix}ivy-local
 Summary:        Local mode for Apache Ivy
 Requires:       %{name} = %{version}-%{release}
-Requires:       javapackages-local = %{version}-%{release}
-Requires:       apache-ivy >= 2.3.0-8
-Requires:       xmvn-connector-ivy >= 2
+Requires:       %{?scl_prefix}javapackages-local = %{version}-%{release}
+Requires:       %{?scl_prefix}apache-ivy >= 2.3.0-8
+Requires:       %{?scl_prefix}xmvn-connector-ivy >= 2
 
-%description -n ivy-local
+%description -n %{?scl_prefix}ivy-local
 This package implements local mode for Apache Ivy, which allows
 artifact resolution using XMvn resolver.
 
-%package -n python3-javapackages
+%package -n %{?scl_prefix}%{python_prefix}-javapackages
 Summary:        Module for handling various files for Java packaging
-Requires:       python3-lxml
-Requires:       python3-six
-Obsoletes:      python-javapackages < %{version}-%{release}
+Requires:       %{python_prefix}-lxml
+Requires:       %{python_prefix}-six
+Obsoletes:      %{?scl_prefix}python-javapackages < %{version}-%{release}
```

Comment:
If `%python_prefix == python` it would obsolete older versions of itself. Not sure if it's a real problem.